bigtable-hbase-2.x-parent/bigtable-hbase-2.x/src/main/java/com/google/cloud/bigtable/hbase2_x/BigtableAsyncAdmin.java

```diff
@@ -515,61 +469,63 @@ private TableName snapshotExists(
         bigtableTableAdminClient.dropAllRowsAsync(tableName.getNameAsString()));
   }
 
-  private Function<List<SnapshotDescription>, Void> deleteSnapshotsFunc() {
-    return snapshots -> {
-      snapshots.stream().forEach(desc -> deleteSnapshot(desc.getName()));
-      return null;
-    };
-  }
-
+  /*
+   * deleteSnapshots is not supported due to backup write quota limits - please use deleteSnapshot
+   */
   @Override
   public CompletableFuture<Void> deleteSnapshots() {
-    return listSnapshots().thenApply(deleteSnapshotsFunc());
+    throw new UnsupportedOperationException("use deleteSnapshot instead");
```

Comment:
Please add a comment saying that quota limits for backup deletion is too strict to support this. So that in the future people dont try to implement this